ui/plugin-system/src/runtime/DataQueriesProvider/model.ts

```diff
@@ -60,26 +60,37 @@ export function transformQueryResults(results: UseQueryResult[], definitions: Qu
 
 export function useQueryType(): (pluginKind: string) => string | undefined {
   const { data: timeSeriesQueryPlugins, isLoading } = useListPluginMetadata('TimeSeriesQuery');
```

Comment:
For the readability, can we rename ``isLoading`` to  ``isTimeSeriesQueryLoading``?

Reply:
Yup, makes sense.